### DISCUSSION TITLE
can't run projects in my local machine

### DISCUSSION THREAD

**Discussion by Prathamesh-Shigwan** [bitfocus/companion#2576] [General]
i want to run and modify companion project on my local machine but i getting errors

PS C:\Users\Administrator\Desktop\test2\companion-master> npm install
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: companion@3.0.1
npm ERR! notsup Not compatible with your version of node/npm: companion@3.0.1
npm ERR! notsup Required: {"npm":"please-use-yarn","yarn":"^1.22","node":">=18.12 <19"}
npm ERR! notsup Actual: {"npm":"10.1.0","node":"v18.17.1"}

what should i do is there any manual that help to run companion-master project on local machines

**Comment by Julusian** [comment]
We use `yarn` instead of `npm`
As it states in this error line

> npm ERR! notsup Required: {"npm":"please-use-yarn","yarn":"^1.22","node":">=18.12 <19"}
